src/Resources/Users/OrderLines.ts

```diff
@@ -0,0 +1,30 @@
+import AbstractResource from '../AbstractResource';
+import { HttpClientConfig, HttpClientResponse } from '../../Interfaces/HttpClient';
+
+export default class OrderLines extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '/order-lines';
+        this.defaultAccess = 'member';
+    }
+
+    public create: Function = this.createMethodFromPartialSpec({
```

Comment:
I'm not certain orderLines is the right name for this, as from the front end point of view, we're adding an orderLine to an order, so may want to make that the semantics for it.

We do the same for baskets and for gifts: https://github.com/whirli/whirli-client

E.G. `$whirli.baskets.addLine()` so `$whirli.orders.addLine()`, rather than `$whirli.orderLines.create()` which is a bit more database led semantics.